portfolio/src/main/webapp/js/TestingReadMe.txt

```diff
@@ -0,0 +1,8 @@
+The tests in custom-script-spec.js require two dependencies I believe
+  The Jasmine framework for the testing.
+    npm install -g jasmine
+  The JSDom library required to utilisie document calls
+    npm install -g jsdom
```

Comment:
Ideally these would get installed via Makefile. Typically you'd write a "make test" clause and have it depend on these packages.

There's not really enough time to work through that, so just take this as an FYI :p

Reply:
Yeah that makes sense